cargo-miri/src/phases.rs

```diff
@@ -541,6 +541,27 @@ pub fn phase_runner(mut binary_args: impl Iterator<Item = String>, phase: Runner
     }
 }
 
+fn get_miriflags() -> Option<Vec<String>> {
```

Comment:
This should probably go into the `util` file.

Reply:
Oh and please add a link to https://doc.rust-lang.org/cargo/reference/config.html#buildrustflags.

I guess we don't support the `target.rustflags` part yet? (That's okay but should be mentioned in a comment.)